.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+  - package-ecosystem: "docker"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "monthly"
```

Comment:
Should we use weekly or monthly interval?

Reply:
I dont have a strong opinion about this one. But in my experience, big number of repos * weekly schedule = hard to review + big number of github action minutes run.

Reply:
monthly is fine